### PR TITLE
add That_Boii to Stackies

### PR DESCRIPTION
**Pull request by D-FOG**

### CODE REVIEW

File: STACKIES.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 | [Onlybond](https://github.com/onlybond) | To improve my managerial skills and my leadership skills with parallely learning about web2 frameworks and web3 concepts |
 | [Raviz](https://github.com/gorvyz)                   | To become a better developer and be healthy and fit       |
 | [Roy.](https://github.com/rywndr)                   | To understand more about programming and get a job in the programming field       |
+| [That_Boii](https://github.com/D-FOG) | To build my finances and coding skills
```
Comment (AbbeyIT) [inline STACKIES.md:12]: Hi @D-FOG you forget one vertical line at the end to close the table. Feel free to edit the file again and commit it.

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

File: STACKIES.md (hunk 1 above)
Comment (D-FOG) [reply STACKIES.md]: Hello @AbbeyIT I have edited it. I think it's resolved now.

**Review by D-FOG — COMMENTED** [review]
@AbbeyIT hello, is it okay

File: STACKIES.md [hunk 2]
```diff
@@ -14,6 +14,7 @@
 | [Raviz](https://github.com/gorvyz)                   | To become a better developer and be healthy and fit       |
 | [Roy.](https://github.com/rywndr)                   | To understand more about programming and get a job in the programming field       |
 | [Ryuint](https://github.com/Ryuint)                   | To make more money for family and happy living       |
+| [That_Boii](https://github.com/D-FOG) | To build my finances and coding skills      |
```
Comment (AbbeyIT) [inline STACKIES.md:17]: LGTM! Thank you for your contribution. 🎉On to more green squares. 🟩

**Review by AbbeyIT — APPROVED** [review]

**Pull request merged**